radixdlt/src/integration/java/com/radixdlt/integration/distributed/simulation/application/LocalMempoolPeriodicSubmittor.java

```diff
@@ -19,24 +19,29 @@
 
 import com.radixdlt.consensus.Command;
 import com.radixdlt.consensus.bft.BFTNode;
+import com.radixdlt.integration.distributed.simulation.SimulationTest.SimulationNetworkActor;
 import com.radixdlt.integration.distributed.simulation.network.SimulationNodes.RunningNetwork;
 import com.radixdlt.mempool.Mempool;
 import com.radixdlt.mempool.MempoolDuplicateException;
 import com.radixdlt.mempool.MempoolFullException;
 import com.radixdlt.utils.Pair;
 import io.reactivex.rxjava3.core.Observable;
+import io.reactivex.rxjava3.disposables.Disposable;
 import io.reactivex.rxjava3.schedulers.Schedulers;
 import io.reactivex.rxjava3.subjects.PublishSubject;
 import java.util.concurrent.TimeUnit;
 
 /**
  * Contributes to steady state by submitting commands to the mempool every few seconds
  */
-public class LocalMempoolPeriodicSubmittor {
+public class LocalMempoolPeriodicSubmittor implements SimulationNetworkActor {
```

Comment:
Seeing as we are editing this, can we fix the spelling here?  Should be `LocalMempoolPeriodicSubmitter`.